superset-frontend/plugins/plugin-chart-cartodiagram/src/util/controlPanelUtil.tsx

```diff
@@ -57,7 +57,7 @@ export const getLayerConfig = (controlPanel: ControlPanelConfig) => {
  */
 export const selectedChartMutator = (
   response: Record<string, any>,
-  value: SelectValue | undefined,
+  value: any,
```

Comment:
Can we use a type here?

Reply:
I should have had a TODO here and other similar places, but the plan is to import all antd-related components from `@superset-ui/core/components` and avoid the `antd` peerDep altogether. I'm happy fast-following with that once the SIP is out. 

The work is going to be easier then with the single flat namespace with all the components and related types.